support-lambdas/stripe-intent/cfn.yaml

```diff
@@ -11,31 +11,6 @@ Parameters:
       - CODE
     Default: CODE
 
-Mappings:
-  StageMap:
-    CODE:
-      DomainName: stripe-intent-code.support.guardianapis.com
-      StripeConfig: test
```

Comment:
it would probably still need to know which stripe config to use

Reply:
good spot!